docs/en/docs/install/source-install/compilation.md

```diff
@@ -35,34 +35,34 @@ This document focuses on how to code Doris through source code.
 
 1. Download Docker Mirror
 
-    `$ docker pull apache/incubator-doris:build-env-ldb-toolchain-latest`
+    `$ docker pull apache/doris:build-env-ldb-toolchain-latest`
 
     Check mirror download completed:
 
     ```
     $ docker images
     REPOSITORY               TAG                              IMAGE ID            CREATED             SIZE
-    apache/incubator-doris   build-env-ldb-toolchain-latest   49f68cecbc1a        4 days ago          3.76GB
+    apache/doris   build-env-ldb-toolchain-latest   49f68cecbc1a        4 days ago          3.76GB
     ```
 
-> Note1: For different versions of Doris, you need to download the corresponding mirror version. From Apache Doris 0.15 version, the docker image will keep same version number with Doris. For example, you can use  `apache/incubator-doris:build-env-for-0.15.0` to compile Apache Doris 0.15.0.
+> Note1: For different versions of Doris, you need to download the corresponding mirror version. From Apache Doris 0.15 version, the docker image will keep same version number with Doris. For example, you can use  `apache/doris:build-env-for-0.15.0` to compile Apache Doris 0.15.0.
 >
-> Node2: `apache/incubator-doris:build-env-ldb-toolchain-latest` is for compiling trunk code, and will be updated along with trunk code. View the update time in `docker/README.md`
+> Node2: `apache/doris:build-env-ldb-toolchain-latest` is for compiling trunk code, and will be updated along with trunk code. View the update time in `docker/README.md`
 
 | image version | commit id | release version |
 |---|---|---|
-| apache/incubator-doris:build-env | before [ff0dd0d](https://github.com/apache/incubator-doris/commit/ff0dd0d2daa588f18b6db56f947e813a56d8ec81) | 0.8.x, 0.9.x |
-| apache/incubator-doris:build-env-1.1 | [ff0dd0d](https://github.com/apache/incubator-doris/commit/ff0dd0d2daa588f18b6db56f947e813a56d8ec81) or later | 0.10.x or later |
-| apache/incubator-doris:build-env-1.2 | [4ef5a8c](https://github.com/apache/incubator-doris/commit/4ef5a8c8560351d7fff7ff8fd51c4c7a75e006a8) or later | 0.12.x - 0.14.0 |
-| apache/incubator-doris:build-env-1.3.1 | [ad67dd3](https://github.com/apache/incubator-doris/commit/ad67dd34a04c1ca960cff38e5b335b30fc7d559f) or later | 0.14.x |
-| apache/incubator-doris:build-env-for-0.15.0 | [a81f4da](https://github.com/apache/incubator-doris/commit/a81f4da4e461a54782a96433b746d07be89e6b54) or later | 0.15.0          |
-| apache/incubator-doris:build-env-latest | before [0efef1b](https://github.com/apache/incubator-doris/commit/0efef1b332300887ee0473f9df9bdd9d7297d824) | |
-| apache/incubator-doris:build-env-for-1.0.0| | 1.0.0 |
-| apache/incubator-doris:build-env-ldb-toolchain-latest | trunk | trunk |
+| apache/incubator-doris:build-env | before [ff0dd0d](https://github.com/apache/doris/commit/ff0dd0d2daa588f18b6db56f947e813a56d8ec81) | 0.8.x, 0.9.x |
```

Comment:
Why not change 'incubator-doris:build-env' to doris:build-env?

Reply:
Some old image has not been moved to the new repo.
And these images are too old, may be removed later.